server/vitals.js

```diff
@@ -280,6 +281,10 @@ async function handleHeartbeat(req, res) {
             await sendIsTamperedAlert(location, isTamperedFlag, mostRecentSensorVitals.isTampered)
           }
 
+          if (isINSZero) {
+            helpers.logSentry(`INS sensor is les or equal zero at ${location.locationid}`)
+          }
+
```

Comment:
Looks good! Just a small spelling mistake with the word "less" - I also wonder if it makes more sense to use "INS value" instead of "INS sensor".

Reply:
Thanks for spotting the typo!